build.gradle

```
@@ -65,6 +65,7 @@ allprojects {
     jcenter()
     mavenCentral()
     maven { url "https://hyperledger-org.bintray.com/besu-repo/" }
+    mavenLocal()
```

We can pull releases and the latest master commit of the discovery project from:
```suggestion
    maven { url "https://bintray.com/consensys/pegasys-repo" }
```

---

Sorry, I gave you the wrong URL, it should be:
`maven { url "https://consensys.bintray.com/pegasys-repo" }`

---

Thanks, that worked